README.md

```diff
@@ -97,27 +97,28 @@ The operator also uses `cert-manager` for issuing certificates to users and brok
 #### Install cert-manager
 
 ```bash
-# pre-create cert-manager namespace and CRDs per their installation instructions
-kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.10.1/deploy/manifests/01-namespace.yaml
+# Create a namespace to run cert-manager in
+kubectl create namespace cert-manager
 ```
```

Comment:
I would delete this since the `https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yam` creates the namespace by default.